src/Serilog.Extensions.Logging/Serilog.Extensions.Logging.csproj

```diff
@@ -35,4 +35,19 @@
     <PackageReference Include="Nullable" Version="1.3.1" PrivateAssets="All" />
   </ItemGroup>
 
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.1' ">
```

Comment:
msbuild conditions do support OR operator :)

Reply:
:-) :+1: 

We flatten them out in the Serilog projects because over time, with a few different conditionals, it becomes easy to miss modifying one when a new target is added (and hard to see at a glance what each target supports).